Webex Connect Flows/v3.0/Template/Event Handling Workflows/README.md

```diff
@@ -0,0 +1,155 @@
+But in this case you need to specify condition for each flow.
+
+As webex.variables and webex.RequestBody are nested variables , we are assigning these to response and requestBody respectively so that we can use them further.
+
+<img width="800" alt="WxCC Task v2 Transition Actions" src="../../images/WxCC Task v2 Transition Actions.png">
+
+##### These event flows are applicable for all channels by default. If v2.x shared flows are already available in the same org, refer to the [Migration Strategy](#migration-strategy) section .
+
+## Adding Conditions In Flow
+If you want to run your flow to be triggered only if the specified conditions are met then it can be achieved by adding conditions.
+You can choose any condition or a set of conditions out of conditions shown in the dropdown of the below image based on your usecase.
+
+<img width="800" alt="ConditionsDropDown" src="../../images/ConditionsDropDown.png">
+
+### Examples of addition conditions in Flow.
+1. If you want to run a flow to run only for a particular channel let's say LiveChat then you can specify condition like webex.mediaChannel equals web.
+
+2. If you want to run a flow only for some specific asset then we can use  the "webex.destination" field which will help us to determine the asset.
+
+Values of "webex.destination" for different channels: -
+* LiveChat :- App ID
+* Whatsapp :- App ID
+* Facebook :- App ID
+* Email :- App ID
+* SMS :- Service Number
+
+The App ID  of a particular asset can be obtained from the assets→apps page in Webex Connect portal.
+
+<img width="800" alt="App" src="../../images/App.png">
+
+The Service Number of a particular sms asset can be obtained from the assets→Numbers page in Webex Connect portal.
+
+<img width="800" alt="App" src="../../images/Numbers.png">
+
+If we want to run the flow for multiple assets then additional new "equals" condition must be added with the OR operator.
+
+<img width="800" alt="OrCondition" src="../../images/ORCondition.png">
+
+If we don't want to run the flow for a specific asset then we can use "notequals" condition.
+
+<img width="800" alt="notequalssinglecondition" src="../../images/notequalssinglecondition.png">
+
+If we don't want to the run the flow for multiple assets then additional "notequals" condition must be added with the AND operator.
+
+<img width="800" alt="NotEquals" src="../../images/NotEquals.png">
+
+For more details on adding conditions please refer https://help.imiconnect.io/docs/start-node#trigger-conditions-in-start-node
+
+### Using variables or details in event flows
+
+Evaluate node is extracting variables from json. 
+In evaluate node we are defining and assigning all the predefined system variables.
+
+<img width="800" alt="Evaluate" src="../../images/Evaluate.png">
+
+We also have helper method to extract flow and global variables which are basically set in the main flows via set variable node.
```

Comment:
Helper method is also provided in the Evaluate node, to extract flow and global variables which are set in the main flows via set variable node.